components/BirthdayListPage/Form.js

```diff
@@ -16,43 +21,73 @@ export default function Form({ onCreateEntry }) {
     };
     onCreateEntry(newEntry);
     event.target.reset();
+    setFormActive(false);
   }
 
   return (
-    <StyledForm onSubmit={handleSubmit}>
-      <StyledInput
-        type="text"
-        placeholder="Name"
-        name="name"
-        aria-label="Name Input"
-        required
-      ></StyledInput>
-      <StyledInput
-        type="text"
-        placeholder="Geschenkideen"
-        name="idea"
-        aria-label="Idea Input"
-      ></StyledInput>
-      <StyledInput
-        type="date"
-        placeholder="Geburtstag"
-        name="date"
-        aria-label="Birthday Input"
-        min="1900-01-01"
-        required
-      />
-      <StyledSubmitButton type="submit" aria-label="Submit Input">
-        Hinzufügen
-      </StyledSubmitButton>
-    </StyledForm>
+    <StyledDiv>
+      {formActive === true ? (
+        <StyledForm onSubmit={handleSubmit}>
+          <StyledNameInput
+            type="text"
+            placeholder="Name"
+            name="name"
+            aria-label="Name Input"
+            className="nameInput"
+            required
+          ></StyledNameInput>
+          <StyledDateInput
+            type="date"
+            placeholder="Geburtstag"
+            name="date"
+            aria-label="Birthday Input"
+            className="dateInput"
+            min="1900-01-01"
+            required
+          />
+          <StyledIdeaInput
+            type="text"
+            placeholder="Geschenkideen"
+            name="idea"
+            aria-label="Idea Input"
+            className="ideaInput"
+          ></StyledIdeaInput>
+          <StyledSubmitButton type="submit" aria-label="Submit Input">
+            Hinzufügen
+          </StyledSubmitButton>
+          <StyledCancelButton
+            onClick={() => {
+              setFormActive(false);
+            }}
+          >
+            <BiXCircle size="20px" />
+          </StyledCancelButton>
+        </StyledForm>
+      ) : (
+        <AddButton
+          onClick={() => {
+            setFormActive(true);
+          }}
+        />
+      )}
+    </StyledDiv>
   );
 }
 
+const StyledDiv = styled.div`
+  display: flex;
+  justify-content: right;
+  position: sticky;
+  bottom: 60px;
+`;
+
 const StyledForm = styled.form`
   position: fixed;
-  display: flex;
-  flex-wrap: wrap;
-  justify-items: center;
+  display: grid;
+  grid-template-areas:
+    "a a b b"
+    "c c c c"
+    "d d d e";
```

Comment:
this is certainly an option to keep the names short, but if you imagine coming back to your code in 3 months, it will be much easier to understand if you renamed the section to something like "Idea",  "date", "submit button"...

Reply:
Good point, I will change it! :) Thanks!